app/controllers/contactController.scala

```diff
@@ -0,0 +1,11 @@
+package controllers
+
+import javax.inject._
+import play.api.mvc._
+
+@Singleton
+class ContactController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+  def contact = Action {
+    Ok(views.html.c("Your new application is ready."))
```

Comment:
Ok(views.html.**c**("Your new application is ready."))
Fix typo ("c" -> "contact")